templates/sonar.properties.erb

```diff
@@ -122,6 +122,18 @@ sonar.web.https.keystoreType=<%= @https['keystoretype'] %>
 sonar.search.javaOpts=<%= @search_java_opts %>
 <% end -%>
 
+<% if @search_port -%>
```

Comment:
@nwheeler Could you please remove  the else statements as default values should be set in `init.pp`?